README.md

```diff
@@ -25,25 +25,46 @@ mbed compile -m XXX -t XXX
 ````
 
 
-# Customer boards
+# Available LoRaWAN modules from distributors
```

Comment:
Note I will add later an additional level:
````
# STM32F1
...
# STM32WL
## Available LoRaWAN modules from distributors
### Seeed Studio LoRa E5
...
## Available Customer boards made with these modules
### Charles's LoRa-E5 breakout board
...
````

Reply:
perfect, like it.